test/web.yml

```diff
@@ -25,8 +25,6 @@ orcidClientSecret: <fill me in>
 gitHubAppPrivateKeyFile: <fill me in>
 gitHubAppId: <fill me in>
 
-orcidClientID: <fill me in>
-orcidClientSecret: <fill me in>
```

Comment:
Why was this removed?

Reply:
oops looks like it was removed by accident. Added it back now.